tests/baselines/reference/moduleAugmentationDeclarationEmit1.js

```diff
@@ -0,0 +1,68 @@
+//// [tests/cases/compiler/moduleAugmentationDeclarationEmit1.ts] ////
+
+//// [map.ts]
+
+import { Observable } from "./observable"
+
+(<any>Observable.prototype).map = function() { }
```

Comment:
I think to some degree it is, since users who have not patched up the prototype prior to this will attempt to do this without asserting to `any`. If we can, let's amend this test to reflect the intended real world usage, or add another test.